dotnet/src/webdriver/Remote/HttpCommandExecutor.cs

```diff
@@ -184,7 +194,7 @@ public virtual Response Execute(Command commandToExecute)
             HttpResponseInfo responseInfo = null;
             try
             {
-                responseInfo = Task.Run(async () => await this.MakeHttpRequest(requestInfo)).GetAwaiter().GetResult();
+                responseInfo = await this.MakeHttpRequest(requestInfo);
```

Comment:
I think it also makes sense to rename existing method `MakeHttpRequest` to `MakeHttpRequestAsync`.